source/libs/utils.js

```diff
@@ -8,10 +8,12 @@ import ghInjection from 'github-injection';
 
 const options = new OptionsSync().getAll();
 
-// `github-injection` happens even when the user navigates in history
-// This causes listeners to run on content that has already been updated.
-// If a feature needs to be disabled when navigating away,
-// use the regular `github-injection`
+/**
+ *`github-injection` happens even when the user navigates in history
+ * This causes listeners to run on content that has already been updated.
+ * If a feature needs to be disabled when navigating away,
+ * use the regular `github-injection`
+ */
```

Comment:
@sindresorhus how about this? https://eslint.org/docs/rules/multiline-comment-style

Reply:
What exactly do you mean? Do you mean enforcing this style for all comments?

Reply:
Yep.

Reply:
It's up to you. I find it slightly inconvenient as you have to manually manage linebreak and formatting. I haven't found a Sublime plugin that automatically handles that. But then again, you write most of the comments here.

Personally, I use single-line comments for simple things:

```js
// Foo
```

And for long comments, this style:

```js
/*
`github-injection` happens even when the user navigates in history
This causes listeners to run on content that has already been updated.
If a feature needs to be disabled when navigating away,
use the regular `github-injection`
*/
```

It's much easier to format.

Reply:
No, I mean a plugin that automatically hard-wraps the comment when written.